apis/management.cattle.io/v3/rke_types.go

```diff
@@ -271,6 +273,23 @@ type KubeAPIService struct {
 	PodSecurityPolicy bool `yaml:"pod_security_policy" json:"podSecurityPolicy,omitempty"`
 	// Enable/Disable AlwaysPullImages admissions plugin
 	AlwaysPullImages bool `yaml:"always_pull_images" json:"alwaysPullImages,omitempty"`
+	// EnableAuditLog to disable logging of audit events
+	EnableAuditLog bool `yaml:"enable_audit_log" json:"enableAuditLog"`
```

Comment:
@leodotcloud do you think we can merge each related 2 fields into a corresponding subtype? Something similar to https://github.com/rancher/types/pull/977/commits/d6bbc84b78a6413f5836c3842b1f2a3acb71d3c7#diff-a0c032a59c1436f31e4912cdc95eb086R275

Reply:
Yes, good suggestion. Will do.